templates/OME-TIFF Conversion.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
<p>Similarly, can we change "FOVs" in the <code>data_dir</code> description to "single-channel TIFFs"? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>